css/page2.css

```diff
@@ -0,0 +1,68 @@
+* {
+  font-family: "Amatic SC", cursive;
+  color: white;
+  background-color: #040f16;
+}
+/* Add style t the div that contain the form */
+.form-popup {
+  /* display: none; */
+  height: 812px;
+}
+
+/* Add style to the form */
+.form-container {
+  padding: 10px;
+}
+
+/* Add style to the input in the form-container form */
+.form-container input[type="text"],
+.form-container input[type="date"],
+.form-container input[type="time"] {
```

Comment:
for every input, remove the outline 

![Screenshot from 2021-07-16 19-27-39](https://user-images.githubusercontent.com/47992412/125979508-7580555b-c30d-4cc4-8845-8475ea6ad95b.png)